hieroplatform/src/main/java/org/hiero/utils/Randomness.java

```diff
@@ -0,0 +1,34 @@
+package org.hiero.utils;
+
+import org.apache.commons.math3.random.MersenneTwister;
+
+public class Randomness {
```

Comment:
Going to static methods is difficult to reverse; so let's stick with the old approach.
The method should be called createInstance().